pages/instructorAssessmentManualGrading/instanceQuestion/gradingPanel.ejs

```diff
@@ -0,0 +1,82 @@
+<form name="manual-grading-form" method="POST"
+      data-max-auto-points="<%= assessment_question.max_auto_points %>"
+      data-max-manual-points="<%= assessment_question.max_manual_points %>"
+      data-max-points="<%= assessment_question.max_points %>">
+  <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+  <input type="hidden" name="modified_at" value="<%= instance_question.modified_at %>">
+  <div class="form-group row justify-content-center">
+    <label class="custom-control-inline col-auto mx-0">
+      <span class="">Points</span>
+      <div class="custom-control custom-switch mx-2">
+        <input class="custom-control-input js-manual-grading-pts-perc-select"
+               name="use_score_perc" type="checkbox">
+        <span class="custom-control-label">Percentage</span>
+      </div>
+    </label>
+  </div>
+  <%- include('gradingPointsSection',
+    { type: 'manual', type_label: 'Manual',
```

Comment:
> How does the manual grading section work for questions that don't have manual grading enabled? Does adding a rubric here enable manual grading? I know I should read the code but I'm being lazy and asking 😅

In this case the question would remain with the same overall number of points, and the manual grading would not affect the auto-grading, but would be an "adjustment" to the auto-grading. For example, if you have a question worth 0 manual points and 20 auto points, and the autograder gave the student 50% (i.e., 10 points), you can have 5 manual points, which would give the question 15/20 overall points, which are 10/20 auto-points and 5/0 manual points. Similarly, you can have a negative number of manual points as a penalty (e.g., for using something that was not allowed but wasn't catched by the auto-grader), so 10/20 auto-points and -5/0 manual points give you 5/20 overall points.

This is already in place since split points were deployed, it's not new in rubrics. If you believe there is a better way to show this I'm open to hear suggestions.